contracts/shapella-upgrade/ShapellaUpgradeTemplate.sol

```diff
@@ -407,67 +377,106 @@ contract ShapellaUpgradeTemplate {
     address public constant _withdrawalQueueImplementation = 0x851f572d3382Ff19ec1f0E04E65B625E32bF21CB;
 
     // Values to set
-    uint256 public constant ACCOUNTING_ORACLE_CONSENSUS_VERSION = 1;
-    string public constant NOR_STAKING_MODULE_NAME = "curated-onchain-v1";
-    bytes32 public constant NODE_OPERATORS_REGISTRY_STAKING_MODULE_TYPE = bytes32("curated-onchain-v1");
-    uint256 public constant NODE_OPERATORS_REGISTRY_STUCK_PENALTY_DELAY = 432000;
-    bytes32 public constant WITHDRAWAL_CREDENTIALS = 0x010000000000000000000000b9d7934878b5fb9610b3fe8a5e441e8fad7e293f;
-    uint256 public constant NOR_STAKING_MODULE_ID = 1;
-    uint256 public constant NOR_STAKING_MODULE_TARGET_SHARE_BP = 10000; // 100%
-    uint256 public constant NOR_STAKING_MODULE_MODULE_FEE_BP = 500; // 5%
-    uint256 public constant NOR_STAKING_MODULE_TREASURY_FEE_BP = 500; // 5%
-    uint256 public constant VALIDATORS_EXIT_BUS_ORACLE_LAST_PROCESSING_REF_SLOT = 0;
-    uint256 public constant VALIDATORS_EXIT_BUS_ORACLE_CONSENSUS_VERSION = 1;
+    uint256 internal constant ACCOUNTING_ORACLE_CONSENSUS_VERSION = 1;
+    string internal constant NOR_STAKING_MODULE_NAME = "curated-onchain-v1";
+    bytes32 internal constant NODE_OPERATORS_REGISTRY_STAKING_MODULE_TYPE = bytes32("curated-onchain-v1");
+    uint256 internal constant NODE_OPERATORS_REGISTRY_STUCK_PENALTY_DELAY = 432000;
+    bytes32 internal constant WITHDRAWAL_CREDENTIALS = 0x010000000000000000000000b9d7934878b5fb9610b3fe8a5e441e8fad7e293f;
+    uint256 internal constant NOR_STAKING_MODULE_ID = 1;
+    uint256 internal constant NOR_STAKING_MODULE_TARGET_SHARE_BP = 10000; // 100%
+    uint256 internal constant NOR_STAKING_MODULE_MODULE_FEE_BP = 500; // 5%
+    uint256 internal constant NOR_STAKING_MODULE_TREASURY_FEE_BP = 500; // 5%
+    uint256 internal constant VALIDATORS_EXIT_BUS_ORACLE_LAST_PROCESSING_REF_SLOT = 0;
+    uint256 internal constant VALIDATORS_EXIT_BUS_ORACLE_CONSENSUS_VERSION = 1;
 
     //
-    // Values for checks to compare with or other
+    // Roles
+    // (stored instead of reading from the contracts to save contract bytecode size)
     //
-    bytes32 public constant DEFAULT_ADMIN_ROLE = 0x00;
-    uint256 public constant NOT_INITIALIZED_CONTRACT_VERSION = 0;
-    uint256 public constant TOTAL_BASIS_POINTS = 10000;
+    bytes32 internal constant DEFAULT_ADMIN_ROLE = 0x00;
```

Comment:
Can we group the roles by the contracts involved? it would be easier to proofread

Reply:
Thanks, fixed